Automattic/simplenote-android
# Set icon size to 48x48 in the monochrome version.

This PR is a small tweak for the monochrome icon to be a bit smaller to 48x48 so that it has some nice padding around the outside.

This is recommended in the guidelines:
https://developer.android.com/develop/ui/views/launch/icon_design_adaptive#design-adaptive-icons

Before:
<img width="349" alt="Screenshot 2024-02-05 at 3 56 50 PM" src="https://github.com/Automattic/simplenote-android/assets/789137/4aabfcf1-8a53-4d04-933d-05a009ec017a">

After: (matches up with Jetpack nicely too!)
<img width="278" alt="Screenshot 2024-02-06 at 2 29 26 PM" src="https://github.com/Automattic/simplenote-android/assets/789137/ed910f04-81b5-41d0-815b-c95a11eca0ef">

**To test**
* Enable color palettes and toggle `Apply palette to app icons`.
* The Simplenote icon should look nice!
